components/Events.js

```diff
@@ -46,6 +64,15 @@ export const Events = [
 		end: new Date(2022, 10, 4, 18, 0, 0),
 		description: "REALLY COOL DESCRIPTION THAT WE NEED TO FIGURE OUT",
 	},
+	{
+		title: "Thinking Like A Programmer",
+		location: "WCH 205/206",
+		color: filters["Meetings"],
+		start: new Date(2022, 10, 8, 16, 0, 0),
+		end: new Date(2022, 10, 8, 18, 0, 0),
+		description:
+			"When getting in computer science, everyone learns a new programming language or how to develop their first web app. But despite all the leetcoding and CS finals, no one wonders how to THINK like a programmer. If you’re interested in learning how to think like one, come to our third GM where we’ll discuss with a professor(TBA) on tips and tricks for thinking like a programmer.",
```

Comment:
Discover the tips and tricks to THINK like a programmer and help you excel in your classes, projects, and various opportunities.